packages/story-editor/src/elements/text/util.js

```diff
@@ -36,6 +36,14 @@ export function generateParagraphTextStyle(
 ) {
   const { font, fontSize, lineHeight, padding, textAlign } = props;
   const { marginOffset } = calcFontMetrics(element);
+
+  const verticalPadding = padding?.vertical || 0;
+  const horizontalPadding = padding?.horizontal || 0;
+  const hasPadding = verticalPadding && horizontalPadding;
```

Comment:
It could also be so that one is `0` and the other not.
```suggestion
  const hasPadding = verticalPadding || horizontalPadding;
```